lib/truffle/truffle/patching.rb

```diff
@@ -78,4 +78,21 @@ def activate
     end
   end
 
+  # Allows specifying patching outside the context of RubyGems by setting an environment variable value.
+  # E.g., setting TRUFFLERUBY_CUSTOM_PATCH='launchy:lib,spec' will allow patching of files in the 'lib' and 'spec'
+  # of a local checkout of the 'launchy' gem.
+  def install_local_patches
+    custom_patch = Truffle.invoke_primitive :java_get_env, 'TRUFFLERUBY_CUSTOM_PATCH' # Use the primitive here rather than ENV so it works if native is disabled.
```

Comment:
`ENV` is not yet available?

Reply:
`ENV` doesn't work if native access is disabled. Since this is called during boot-up, that's a case we need to handle.